.circleci/config.yml

```diff
@@ -23,7 +23,7 @@ jobs:
             #       command: npm set prefix=/home/circleci/npm && echo 'export PATH=$HOME/circleci/npm/bin:$PATH' >> /home/circleci/.bashrc
             - run:
                   name: install-yarn
-                  command: npm install --global yarn@1.17.0
+                  command: npm install --global yarn@1.17.0 || echo "Skipping install"
```

Comment:
Temporarily changed to get circle to work. Will not get merged. See issue [here](https://github.com/yarnpkg/yarn/issues/7773).